src/game/client/swarm/vgui/nb_mission_summary.cpp

```diff
@@ -139,6 +141,26 @@ void CNB_Mission_Summary::OnThink()
 		m_pMissionLabel->SetText(pMap->m_szMissionTitle);
 	}
 
+	int nSpeedrunTime = CAlienSwarm::GetSpeedrunTime();
+	wchar_t wszSRTime[ 128 ];
+
+	if ( nSpeedrunTime <= 0 )
+		V_snwprintf( wszSRTime, ARRAYSIZE( wszSRTime ), L"-" );
+	else
+		V_snwprintf( wszSRTime, ARRAYSIZE( wszSRTime ), g_pVGuiLocalize->FindSafe( "#nb_speedruntime_format" ), nSpeedrunTime / 60, nSpeedrunTime % 60 / 10, nSpeedrunTime % 10 );
+
+	m_pSpeedrunTimeLabel->SetText( wszSRTime );
+
+	int nOEStatus = CAlienSwarm::GetOutstandingExecutionStatus();
+	wchar_t wszOEStatus[ 128 ];
+
+	if ( nOEStatus < 0 )
+		V_snwprintf( wszOEStatus, ARRAYSIZE( wszOEStatus ), L"-" );
+	else
+		V_snwprintf( wszOEStatus, ARRAYSIZE( wszOEStatus ), L"%s", nOEStatus ? g_pVGuiLocalize->FindSafe( "#nb_oe_status_active" ) : g_pVGuiLocalize->FindSafe( "#nb_oe_status_failed" ) );
```

Comment:
since there's only one string here, we can just use the result of `g_pVGuiLocalize->Find` directly